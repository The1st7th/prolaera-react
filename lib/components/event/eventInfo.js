'use strict';

Object.defineProperty(exports, '__esModule', {
  value: true
});

var _createClass = (function() {
  function defineProperties(target, props) {
    for (var i = 0; i < props.length; i++) {
      var descriptor = props[i];
      descriptor.enumerable = descriptor.enumerable || false;
      descriptor.configurable = true;
      if ('value' in descriptor) descriptor.writable = true;
      Object.defineProperty(target, descriptor.key, descriptor);
    }
  }
  return function(Constructor, protoProps, staticProps) {
    if (protoProps) defineProperties(Constructor.prototype, protoProps);
    if (staticProps) defineProperties(Constructor, staticProps);
    return Constructor;
  };
})();

var _react = require('../../../../../Library/Caches/typescript/2.9/node_modules/@types/react');

var _react2 = _interopRequireDefault(_react);

var _reactHtmlEmail = require('react-html-email');

var _dateHelpers = require('../../helpers/dateHelpers');

var _deliveryHelper = require('../../helpers/deliveryHelper');

var _deliveryHelper2 = _interopRequireDefault(_deliveryHelper);

var _domHelpers = require('../../helpers/domHelpers');

var _domHelpers2 = _interopRequireDefault(_domHelpers);

var _button = require('../button');

var _button2 = _interopRequireDefault(_button);

function _interopRequireDefault(obj) {
  return obj && obj.__esModule ? obj : { default: obj };
}

function _classCallCheck(instance, Constructor) {
  if (!(instance instanceof Constructor)) {
    throw new TypeError('Cannot call a class as a function');
  }
}

function _possibleConstructorReturn(self, call) {
  if (!self) {
    throw new ReferenceError("this hasn't been initialised - super() hasn't been called");
  }
  return call && (typeof call === 'object' || typeof call === 'function') ? call : self;
}

function _inherits(subClass, superClass) {
  if (typeof superClass !== 'function' && superClass !== null) {
    throw new TypeError('Super expression must either be null or a function, not ' + typeof superClass);
  }
  subClass.prototype = Object.create(superClass && superClass.prototype, {
    constructor: { value: subClass, enumerable: false, writable: true, configurable: true }
  });
  if (superClass)
    Object.setPrototypeOf ? Object.setPrototypeOf(subClass, superClass) : (subClass.__proto__ = superClass);
}

var EventInfo = (function(_React$Component) {
  _inherits(EventInfo, _React$Component);

  function EventInfo() {
    _classCallCheck(this, EventInfo);

    return _possibleConstructorReturn(
      this,
      (EventInfo.__proto__ || Object.getPrototypeOf(EventInfo)).apply(this, arguments)
    );
  }

  _createClass(EventInfo, [
    {
      key: 'render',
      value: function render() {
        var _props = this.props,
          buttonText = _props.buttonText,
          event_id = _props.event_id,
          _props$course_name = _props.course_name,
          course_name = _props$course_name === undefined ? '' : _props$course_name,
          _props$buttonLink = _props.buttonLink,
          buttonLink =
            _props$buttonLink === undefined
              ? 'https://app.prolaera.com/#/events/' + ('' + event_id)
              : _props$buttonLink,
          _props$delivery_date = _props.delivery_date,
          delivery_date = _props$delivery_date === undefined ? '' : _props$delivery_date,
          _props$delivery_end_d = _props.delivery_end_date,
          delivery_end_date = _props$delivery_end_d === undefined ? '' : _props$delivery_end_d,
          _props$hours = _props.hours,
          hours = _props$hours === undefined ? [] : _props$hours,
          _props$delivery_locat = _props.delivery_location,
          delivery_location = _props$delivery_locat === undefined ? '' : _props$delivery_locat,
          _props$delivery_metho = _props.delivery_method,
          delivery_method = _props$delivery_metho === undefined ? 1 : _props$delivery_metho,
          _props$price = _props.price,
          price = _props$price === undefined ? 0 : _props$price,
          _props$courseAudience = _props.courseAudience,
          courseAudience = _props$courseAudience === undefined ? '' : _props$courseAudience,
          _props$prep = _props.prep,
          prep = _props$prep === undefined ? '' : _props$prep,
          _props$prerequisites = _props.prerequisites,
          prerequisites = _props$prerequisites === undefined ? '' : _props$prerequisites,
          _props$level = _props.level,
          level = _props$level === undefined ? '' : _props$level,
          _props$objectives = _props.objectives,
          objectives = _props$objectives === undefined ? '' : _props$objectives,
          _props$summary = _props.summary,
          summary = _props$summary === undefined ? '' : _props$summary;

        return _react2.default.createElement(
          'div',
          {
            style: {
              display: 'block',
              marginLeft: 'auto',
              marginRight: 'auto',
              marginBottom: '20px',
              textAlign: 'center',
              borderRadius: '5px',
              maxWidth: '500px',
              padding: '20px',
              fontFamily: 'Arial, sans-serif',
              backgroundColor: 'white'
            }
          },
          _react2.default.createElement(
            _reactHtmlEmail.Box,
            { className: 'eventInfo', align: 'center', width: '100%' },
            _react2.default.createElement(
              _reactHtmlEmail.Item,
              { align: 'left' },
              _react2.default.createElement(
                'h3',
                { align: 'center', style: { marginTop: '0px' } },
                'Event Information:'
              ),
              _react2.default.createElement(
                'h4',
                { align: 'center' },
                _react2.default.createElement(
                  'em',
                  null,
                  course_name,
                  ' (',
                  (0, _deliveryHelper2.default)(delivery_method),
                  ')'
                )
              )
            ),
            _react2.default.createElement(
              _reactHtmlEmail.Item,
              { align: 'left' },
              _react2.default.createElement(
                'div',
                { style: { paddingLeft: '30px' } },
                _react2.default.createElement(
                  'h4',
                  null,
                  'Start Time: ',
                  _react2.default.createElement(
                    'span',
                    { style: { fontWeight: 'normal' } },
                    (0, _dateHelpers.PrettyDate)(delivery_date)
                  )
                ),
                _react2.default.createElement(
                  'h4',
                  null,
                  'End Time: ',
                  _react2.default.createElement(
                    'span',
                    { style: { fontWeight: 'normal' } },
                    (0, _dateHelpers.PrettyDate)(delivery_end_date)
                  )
                ),
                _react2.default.createElement('h4', { style: { marginBottom: '0px' } }, 'Recommended CPE Credit(s):'),
                _react2.default.createElement(
                  'ul',
                  { style: { marginTop: '10px' } },
                  hours.map(function(hour, index) {
                    return _react2.default.createElement(
                      'li',
                      { key: index, style: { padding: '5px' } },
                      hour.subject_area,
                      ' - ',
                      hour.credits,
                      ' Hour(s)'
                    );
                  })
                ),
                _react2.default.createElement(
                  'h4',
                  null,
                  'Location:',
                  _react2.default.createElement(
                    'a',
                    { href: 'https://www.google.com/maps/place/' + ('' + delivery_location) },
                    _react2.default.createElement('span', { style: { fontWeight: 'normal' } }, delivery_location)
                  )
                ),
                _react2.default.createElement(
                  'h4',
                  null,
                  'Delivery Method: ',
                  _react2.default.createElement(
                    'span',
                    { style: { fontWeight: 'normal' } },
                    (0, _deliveryHelper2.default)(delivery_method)
                  )
                ),
                _react2.default.createElement(
                  'h4',
                  null,
                  'Price: ',
                  _react2.default.createElement('span', { style: { fontWeight: 'normal' } }, '$', price, '.00')
                ),
                _react2.default.createElement(
                  'h4',
                  null,
                  'Target Audience: ',
                  _react2.default.createElement('span', { style: { fontWeight: 'normal' } }, courseAudience)
                ),
                _react2.default.createElement('h4', { style: { marginBottom: '0px' } }, 'Prep:'),
                _react2.default.createElement('div', {
                  className: 'innerHtmlStyles',
                  dangerouslySetInnerHTML: (0, _domHelpers2.default)(prep)
                }),
                _react2.default.createElement('h4', { style: { marginBottom: '0px' } }, 'Prerequisites:'),
                _react2.default.createElement('div', {
                  className: 'innerHtmlStyles',
                  dangerouslySetInnerHTML: (0, _domHelpers2.default)(prerequisites)
                }),
                _react2.default.createElement(
                  'h4',
                  null,
                  'Level: ',
                  _react2.default.createElement('span', { style: { fontWeight: 'normal' } }, level)
                ),
                _react2.default.createElement('h4', { style: { marginBottom: '0px' } }, 'Learning Objectives:'),
                _react2.default.createElement('div', {
                  className: 'innerHtmlStyles',
                  dangerouslySetInnerHTML: (0, _domHelpers2.default)(objectives)
                }),
                _react2.default.createElement('h4', { style: { marginBottom: '0px' } }, 'Summary:'),
                _react2.default.createElement('div', {
                  className: 'innerHtmlStyles',
                  dangerouslySetInnerHTML: (0, _domHelpers2.default)(summary)
                })
              )
            ),
            _react2.default.createElement(
              _reactHtmlEmail.Item,
              { align: 'center' },
              _react2.default.createElement(
                'div',
                null,
                buttonText
                  ? _react2.default.createElement(_button2.default, { text: buttonText, link: buttonLink })
                  : _react2.default.createElement('span', { className: 'buttonSpan' })
              )
            )
          )
        );
      }
    }
  ]);

  return EventInfo;
})(_react2.default.Component);

exports.default = EventInfo;
