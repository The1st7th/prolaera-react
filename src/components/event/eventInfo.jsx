import React from 'react';
import { Box, Item } from 'react-html-email';
import { PrettyDate } from '../../helpers/dateHelpers';
import deliveryHelper from '../../helpers/deliveryHelper';
import setInnerHtml from '../../helpers/domHelpers';
import Button from '../button';

class EventInfo extends React.Component {
  render() {
    const {
      buttonText,
      event_id,
      course_name = '',
      buttonLink = 'https://app.prolaera.com/#/events/' + `${event_id}`,
      delivery_date = '',
      delivery_end_date = '',
      hours = [],
      delivery_location = '',
      delivery_method = 1,
      price = 0,
      courseAudience = '',
      prep = '',
      prerequisites = '',
      level = '',
      objectives = '',
      summary = ''
    } = this.props;

    let newPrerequisites = prerequisites.trim();

    return (
      <div
        id="subHeaderWrapper"
        style={{
          display: 'block',
          marginLeft: 'auto',
          marginRight: 'auto',
          marginBottom: '0px',
          textAlign: 'center',
          padding: '20px',
          backgroundColor: '#FFFFFF'
        }}
      >
        <Box className="eventInfo" align="center" width="100%">
          <Item align="center" style={{ paddingBottom: '20px' }} />
          <Item align="left">
            <div
              style={{
                width: '90%',
                height: '20px',
                borderBottom: '1px solid grey',
                textAlign: 'center',
                boxSizing: 'border-box',
                position: 'relative',
                margin: 'auto',
                padding: '10px'
              }}
            >
              <span
                style={{
                  fontSize: '16pt',
                  backgroundColor: '#FFFFFF',
                  padding: '0px 20px',
                  fontWeight: 'lighter',
                  boxSizing: 'border-box',
                  margin: '0'
                }}
              >
                Event Information
              </span>
            </div>
            {/* <h3 className="infoHeader" align="center" style={{ marginTop: '0px' }}>
              Event Information:
            </h3> */}
            <h4 align="center" style={{ paddingTop: '20px' }}>
              <em>
                {course_name} ({deliveryHelper(delivery_method)})
              </em>
            </h4>
          </Item>
          <Item align="left">
            <div style={{ paddingLeft: '30px' }}>
              <h4>Start Time:</h4>
              <div
                className="innerHtmlStyles"
                style={{ fontWeight: 'normal' }}
                dangerouslySetInnerHTML={setInnerHtml(PrettyDate(delivery_date))}
              />
              <h4>End Time:</h4>
              <div
                className="innerHtmlStyles"
                style={{ fontWeight: 'normal' }}
                dangerouslySetInnerHTML={setInnerHtml(PrettyDate(delivery_end_date))}
              />
              <h4 style={{ marginBottom: '0px' }}>Recommended CPE Credit(s):</h4>
              <ul style={{ marginTop: '10px' }}>
                {hours.map((hour, index) => (
                  <li key={index} style={{ padding: '5px' }}>
                    {hour.subject_area} - {hour.credits} Hour(s)
                  </li>
                ))}
              </ul>
              <h4>Location:</h4>
              <div className="innerHtmlStyles" style={{ fontWeight: 'normal' }}>
                <a
                  style={{ textDecoration: 'underline', color: 'inherit' }}
                  href={'https://www.google.com/maps/place/' + `${delivery_location}`}
                >
                  <span style={{ fontWeight: 'normal' }}>{delivery_location}</span>
                </a>
              </div>
              <h4>Delivery Method:</h4>
              <div
                className="innerHtmlStyles"
                style={{ fontWeight: 'normal' }}
                dangerouslySetInnerHTML={setInnerHtml(deliveryHelper(delivery_method))}
              />
              <h4>Price: </h4>
              <div
                className="innerHtmlStyles"
                style={{ fontWeight: 'normal' }}
                dangerouslySetInnerHTML={setInnerHtml('$' + price + '.00')}
              />
              <h4>Target Audience:</h4>
              <div
                className="innerHtmlStyles"
                style={{ fontWeight: 'normal' }}
                dangerouslySetInnerHTML={setInnerHtml(courseAudience)}
              />
              <h4 style={{ marginBottom: '0px' }}>Prep:</h4>
              <div
                className="innerHtmlStyles"
                style={{ fontWeight: 'normal' }}
                dangerouslySetInnerHTML={setInnerHtml(prep)}
              />
              <h4 style={{ marginBottom: '0px' }}>Prerequisites:</h4>
              <div
                className="innerHtmlStyles"
                style={{ fontWeight: 'normal' }}
                dangerouslySetInnerHTML={setInnerHtml(newPrerequisites)}
              />
              <h4>Level:</h4>
              <div
                className="innerHtmlStyles"
                style={{ fontWeight: 'normal' }}
                dangerouslySetInnerHTML={setInnerHtml(level)}
              />
              <h4 style={{ marginBottom: '0px' }}>Learning Objectives:</h4>
              <div
                className="innerHtmlStyles"
                style={{ fontWeight: 'norma;' }}
                dangerouslySetInnerHTML={setInnerHtml(objectives)}
              />
              <h4 style={{ marginBottom: '0px' }}>Summary:</h4>
              <div
                className="innerHtmlStyles"
                style={{ fontWeight: 'normal' }}
                dangerouslySetInnerHTML={setInnerHtml(summary)}
              />
            </div>
          </Item>
          <Item align="center">
            {buttonText ? (
              <div
                style={{
                  margin: 'auto',
                  textAlign: 'middle'
                }}
              >
                {buttonText ? (
                  <Button color={'#72C02C'} text={buttonText} link={buttonLink} />
                ) : (
                  <span className="buttonSpan" />
                )}
              </div>
            ) : (
              <span color={'#72C02C'} className="buttonSpan" />
            )}
          </Item>
        </Box>
      </div>
    );
  }
}

export default EventInfo;
