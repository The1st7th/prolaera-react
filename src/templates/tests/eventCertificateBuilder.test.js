import renderer from 'react-test-renderer';
import writeFile from '../../helpers/writeFileHelper';
import user from '../../templates/tests/json/completeUser.json';
import event from '../../templates/tests/json/event.json';
import eventCertificateEmail from '../builders/eventCertificateBuilder';
import testCertificate from '../../templates/tests/json/completeCertificate.json';
import inlineCss from 'inline-css';
var request = require('request');

const _sendEmail = (template, emails) => {
  return new Promise((resolve, reject) => {
    request(
      {
        url: 'https://api-dev.prolaera.com/v1/mailer',
        method: 'PUT',
        json: {
          template,
          emails
        }
      },
      (error, response, body) => {
        if (error) {
          console.error(error);
          return reject(error);
        }
        resolve({
          response,
          body
        });
      }
    );
  });
};
describe('eventCertificateBuilder', () => {
  const logoUrl = 'https://assets.prolaera.com/prolaeraLogo_fullText.png';

  it('checks snapshot', async () => {
    const emailHtml = await eventCertificateEmail(event, user, testCertificate, logoUrl);
    const emailComponent = renderer.create(emailHtml);
    const registrationJson = emailComponent.toJSON();
    expect(registrationJson).toMatchSnapshot();
  });

  it('writes an HTML file', async () => {
    const email = await eventCertificateEmail(event, user, testCertificate, logoUrl);
    const newEmail = await inlineCss(email, {
      url: ' ',
      preserveMediaQueries: true,
      applyWidthAttributes: true,
      applyTableAttributes: true
    });
    await _sendEmail(newEmail, ['eric.e.nicolas@gmail.com']);
    const saved = await writeFile(newEmail, 'eventCertificateTest.html');
    expect(saved).toEqual(true);
  });
});
