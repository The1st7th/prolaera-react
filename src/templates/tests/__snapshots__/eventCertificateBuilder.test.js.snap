// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`eventCertificateBuilder checks snapshot 1`] = `
"<!DOCTYPE html PUBLIC \\"-//W3C//DTD XHTML 1.0 Strict//EN\\" \\"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd\\"><html lang=\\"en\\" xmlns=\\"http://www.w3.org/1999/xhtml\\"><head><meta http-equiv=\\"Content-Type\\" content=\\"text/html; charset=utf-8\\"/><meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1.0\\"/><title>You have completed an event</title><style type=\\"text/css\\">body {
	background-color: #F7F7F7;
	font-family: source-sans-pro, sans-serif;
}

#subHeaderWrapper {
	padding: 20px;
}

.innerHtmlStyles {
	margin-top: 10px;
}
.innerHtmlStyles p, .innerHtmlStyles ol, .innerHtmlStyles ul  {
	margin-top: 10px;
}

@media
only screen and (max-width: 600px),
(min-device-width: 600px) and (max-device-width: 1024px)  {

	/* Force table to not be like tables anymore */
	table, thead, tbody, th, td, tr {
		display: block;
	}

	.emailHeader td {
		width: 100%;
	}

	.emailHeader td img {
		margin-left: auto;
		margin-right: auto;
	} 

	.emailFooter {
		width: 100%;
		color: #F7F7F7;
		margin-right: auto;
		margin-left: auto;
		font-size: 10pt;
	}
	.emailFooter td {
		padding: 0px 10px 5px 10px;
		font-size: 10pt;
	}

	#subHeaderWrapper {
		width: 100%;
		padding: 0px;
	}

	.report td {
		/* Behave  like a &quot;row&quot; */
		border: none;
		border-bottom: 1px solid #eee;
    	position: relative;
    	text-align: center;
	}

	.report td:before {
		/* Now like a table header */
		position: absolute;
		/* Top/left values mimic padding */
		top: 6px;
		left: 6px;
		width: 45%;
		padding-right: 10px;
		white-space: nowrap;
	}
	
	.customButton:hover {
		color:black;
		
	}


/*End of media query*/
}</style></head><body style=\\"width:100%;margin:0;padding:0;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%\\"><table width=\\"100%\\" height=\\"100%\\" cellPadding=\\"0\\" cellSpacing=\\"0\\" border=\\"0\\" align=\\"left\\" valign=\\"top\\"><tbody><tr><td align=\\"center\\" valign=\\"top\\"><table width=\\"600\\" align=\\"center\\" cellPadding=\\"0\\" cellSpacing=\\"0\\" border=\\"0\\" valign=\\"top\\"><tbody><table class=\\"emailHeader\\" align=\\"center\\" width=\\"100%\\" style=\\"background-color:#F7F7F7\\" cellPadding=\\"0\\" cellSpacing=\\"0\\" border=\\"0\\" valign=\\"top\\"><tbody><tr><td align=\\"center\\"><table align=\\"center\\" cellSpacing=\\"0\\" style=\\"padding:15px;max-width:584px;margin-top:25px;margin-bottom:20px\\"><tr align=\\"center\\"><td width=\\"292px\\"><img alt=\\"logo\\" src=\\"https://assets.prolaera.com/prolaeraLogo_fullText.png\\" width=\\"199.00305623471883\\" height=\\"47\\" style=\\"display:block;outline:none;border:none;text-decoration:none\\"/></td></tr></table></td></tr></tbody></table><div id=\\"subHeaderWrapper\\" style=\\"display:block;margin-left:auto;margin-right:auto;margin-top:20px;margin-bottom:20px;max-width:500px;font-family:Arial, sans-serif;border-radius:5px;background-color:white\\"><table class=\\"subHeader\\" align=\\"center\\" width=\\"100%\\" cellPadding=\\"0\\" cellSpacing=\\"0\\" border=\\"0\\" valign=\\"top\\"><tbody><tr><td><table id=\\"subHeaderTable\\" style=\\"text-align:center;margin-left:auto;margin-right:auto\\"><tr><td><p style=\\"color:gray;font-style:italic;margin-bottom:5px\\">You have received a</p></td></tr><tr><td><h2 style=\\"font-size:35px;margin-top:0px;margin-bottom:5px\\">Certificate</h2></td></tr><tr><td><p></p><div><p>Congratulations John! You&#x27;ve been issued a new certificate of completion. Please take a few moments to complete an evaluation to help us improve by clicking the Complete Evaluation button below.</p></div></td></tr></table></td></tr></tbody></table></div><div style=\\"display:block;margin-left:auto;margin-right:auto;margin-bottom:20px;text-align:center;border-radius:5px;max-width:500px;padding:20px;font-family:Arial, sans-serif;background-color:white\\"><table align=\\"center\\" width=\\"100%\\" cellPadding=\\"0\\" cellSpacing=\\"0\\" border=\\"0\\" valign=\\"top\\"><tbody><tr><td><div style=\\"max-width:300px;margin-left:auto;margin-right:auto;border:1px solid lightgray;border-radius:8px;background:linear-gradient(to bottom right, #ffd11a, #fff0b3, #ffd633);box-shadow:1px 1px 8px darkgray\\"><h3 style=\\"margin-bottom:5px\\">ASC 606 Update Training</h3><h6 style=\\"margin-top:0px\\">July 18, 2018 - July 19, 2018</h6></div></td></tr><tr><td><p></p></td></tr><tr><td><div class=\\"customButton\\" align=\\"center\\" style=\\"max-width:100%;text-align:center;padding:20px 0px 20px 0px\\"><a href=\\"https://files.prolaera.com.s3-us-west-2.amazonaws.com/companies/57/users/0d0f34ac-7ce3-41a4-9231-5cf8b52a5ed5/certificates/0002bd45-0575-4856-b09e-59c8c54c5e20.pdf\\" style=\\"width:200px;padding:14px 14px 14px 14px;color:white;text-decoration:none;border-radius:3px\\">View Certificate</a></div><hr style=\\"width:45px\\"/><div class=\\"customButton\\" align=\\"center\\" style=\\"max-width:100%;text-align:center;padding:20px 0px 20px 0px\\"><a href=\\"https://app.prolaera.com/#/users/0d0f34ac-7ce3-41a4-9231-5cf8b52a5ed5/events/d342572d-e149-48fa-83c5-e2730bbb1b76/review\\" style=\\"width:200px;padding:14px 14px 14px 14px;color:white;text-decoration:none;border-radius:3px\\">Complete Evaluation</a></div></td></tr></tbody></table></div><div style=\\"display:block;margin-left:auto;margin-right:auto;margin-bottom:20px;text-align:center;border-radius:5px;max-width:500px;padding:20px;font-family:Arial, sans-serif;background-color:white\\"><table class=\\"eventInfo\\" align=\\"center\\" width=\\"100%\\" cellPadding=\\"0\\" cellSpacing=\\"0\\" border=\\"0\\" valign=\\"top\\"><tbody><tr><td align=\\"left\\"><h3 align=\\"center\\" style=\\"margin-top:0px\\">Event Information:</h3><h4 align=\\"center\\"><em>ASC 606 Update Training (Group-Live)</em></h4></td></tr><tr><td align=\\"left\\"><div style=\\"padding-left:30px\\"><h4>Start Time: <span style=\\"font-weight:normal\\">July 18, 2018</span></h4><h4>End Time: <span style=\\"font-weight:normal\\">July 19, 2018</span></h4><h4 style=\\"margin-bottom:0px\\">Recommended CPE Credit(s):</h4><ul style=\\"margin-top:10px\\"><li style=\\"padding:5px\\">Accounting - 2 Hour(s)</li></ul><h4>Location:<a href=\\"https://www.google.com/maps/place/111 S Jackson St\\"><span style=\\"font-weight:normal\\">111 S Jackson St</span></a></h4><h4>Delivery Method: <span style=\\"font-weight:normal\\">Group-Live</span></h4><h4>Price: <span style=\\"font-weight:normal\\">$0.00</span></h4><h4>Target Audience: <span style=\\"font-weight:normal\\">CPAs</span></h4><h4 style=\\"margin-bottom:0px\\">Prep:</h4><div class=\\"innerHtmlStyles\\"><p>None</p></div><h4 style=\\"margin-bottom:0px\\">Prerequisites:</h4><div class=\\"innerHtmlStyles\\"><p>&nbsp;None&nbsp;<br></p></div><h4>Level: <span style=\\"font-weight:normal\\">Basic</span></h4><h4 style=\\"margin-bottom:0px\\">Learning Objectives:</h4><div class=\\"innerHtmlStyles\\"><ol><li>One</li><li>Two</li><li>Three</li></ol></div><h4 style=\\"margin-bottom:0px\\">Summary:</h4><div class=\\"innerHtmlStyles\\">This is a course summary.</div></div></td></tr><tr><td align=\\"center\\"><div><span class=\\"buttonSpan\\"></span></div></td></tr></tbody></table></div><table class=\\"emailFooter\\" align=\\"center\\" width=\\"540px\\" cellSpacing=\\"20\\" style=\\"background-color:#333333\\" cellPadding=\\"0\\" border=\\"0\\" valign=\\"top\\"><tbody><tr><td><td style=\\"float:left\\"><span style=\\"font-family:source-sans-pro, sans-serif;font-size:10pt;font-weight:100;line-height:14px;color:#F7F7F7;letter-spacing:1pt;float:left;padding:10px 5px 5px 0px\\">Powered By</span><img alt=\\"react\\" src=\\"https://gallery.mailchimp.com/eb930e30a15249a18d658c183/images/196fa069-3113-464c-a304-a1aa9f99fe22.png\\" width=\\"100\\" height=\\"23\\" style=\\"display:block;outline:none;border:none;text-decoration:none;padding-top:5px\\"/></td><td style=\\"font-family:source-sans-pro, sans-serif;color:#F7F7F7;font-size:10pt;font-weight:100;letter-spacing:1pt;text-align:right;padding-top:10px;padding-right:10px;padding-bottom:10px\\">Â© Prolaera 2018</td></td></tr></tbody></table></tbody></table></td></tr></tbody></table></body></html>"
`;
